packages/react/src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -209,6 +209,8 @@ export default class FocusTrapZone extends React.Component<FocusTrapZoneProps, {
     if (!this._root.current.contains(relatedTarget as HTMLElement)) {
       this._hasFocus = false
     }
+
+    ev.stopPropagation()
```

Comment:
Without this, tabbing inside a `Popup` that was created as part of a `ToolbarMenu`, will cause the `Popup` to be caused. The `onBlur` event in `ToolbarItem` would be called, which I believe is not correct behavior (since the `Popup` is in a focus trap).

Reply:
**This also prevents an issue, where you click on the Popup's background and it gets closed.**